terminus-ui/scss/helpers/_layout.scss

```diff
@@ -35,7 +35,7 @@
   $position: absolute,
   $content: ''
 ) {
-  content: $content;
+  content: unquote("\'#{$content}\'");
```

Comment:
This is an interesting way to escape...

Reply:
We had to do this because our linter was changing double quote to single quote. There were other linting issues, but I assume we'll iron those out.

The function is built in to SASS: https://sass-lang.com/documentation/functions/string